docs/ReleaseNotes.md

```diff
@@ -19,7 +19,7 @@ The Guava dependency version has been updated to 31.1. Projects may need to chec
 * **Bug fix** Fix 2 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Bug fix** Fix 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Bug fix** Fix 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Bug fix** Fix 5 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Bug fix** The `EMPTY_SCANS` metric is no double counts scans that call `hasNext` multiple times after completing an empty range scan [(Issue #2002)](https://github.com/FoundationDB/fdb-record-layer/issues/2002)
```

Comment:
This doesn't seem to have proper grammar.

Reply:
Whoops, yeah, I got two different versions of this comment jumbled as I was writing it.